### ISSUE TITLE
Make main menu translatable

### ISSUE THREAD

**Issue by Quintus** [Secretchronicles/TSC#244]
The main menu currently consists of images. This makes it impossible to localise it, hence the images have to go away.

This task depends on #10, as rewriting GUI stuff has to wait until our discussions regarding CEGUI have come to something.

Vale,
Quintus

**Comment by Quintus** [comment]
Superseded by #407.
